### PR TITLE
15 create portfolio popup window

### LINKED ISSUE #15: create-portfolio-popup-window
### Issue: 14-create-portfolio-popup-window

-  A mobile popup is created.
- A desktop popup is created.
- A JavaScript Object  to store all the information about the projects is created.
- The project section will be created dynamically.
- Linters are going to be added to the project.
- Figma design tool is used.

### Goal:
- Use JavaScript syntax and access DOM elements.
- Understand and use JavaScript Events.
- Understand how to parse a Figma design to create a UI.
- Use objects to store and access data.

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About this project 📍
- This project is about the creation of a **Popup Window** for the mobile and desktop version of the project.

- A new feature branch was created to work specific in this popup functionality.
- A file named menu.js was created to add the functionality of the popup window.
- Also in index.html the section named works has only a section tag and the elements placed there were erased.
- These elements are created in the menu.js file.
- The following elements were created there: an object, variables accessed by DOM elements, events, loops, add method and events.
- Figma design tool was used to accomplish this task.
- CSS Styles are added to the items created in the menu.js file.
- Tools added: Stylelint, Webhint and ESLint. They are important to deliver a clean code in HTML, CSS, and JavaScript.

### why these tools are important? 📍
Figma, Webhint, Stylelint, ESLInt, JavaScript Events, loops, object and JavaScript DOM are essential to create a popup window.

### what was the most complex part? ⭐️
I feel that understand how to give the functionality to the popup window and also the usage of Events and objects were the hardest things to me. I appreciate this learning because it gives me news tools to interact with and deliver dynamic designs.

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @IvonneBenitesRodriguez 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have collaboration from your coding partner. kindly contact them so you can work together and make some commits.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/menu.js [hunk 1]
```diff
@@ -20,3 +20,169 @@ hamburgerIcon.addEventListener('click', appear);
...
+
+for (let i = 0; i < projectsData.length; i += 1) {
+  document.querySelector(`.project-modal-${i}`).addEventListener('click', () => {
+    projectName.innerHTML = projectsData[i].name;
+    modalImg.src = projectsData[i].imageLink;
+    company.innerHTML = projectsData[i].company;
+    role.innerHTML = projectsData[i].role;
+    year.innerHTML = projectsData[i].year;
+
+    modal.innerHTML = `
```
Comment (Ismailco) [inline js/menu.js:132]: - [ ] On the desktop view, I can't scroll down to see the modal information and buttons, kindly make sure to fix this so the modal fits desktop sizes properly

Screenshot_20230729_162816

File: js/menu.js [hunk 2]
```diff
@@ -20,3 +20,169 @@ hamburgerIcon.addEventListener('click', appear);
...
+];
+
+workSections.innerHTML = '';
+for (let i = 0; i < projectsData.length; i += 1) {
+  // let languages = '';
+  // projectsData[i].technologies.forEach((language) => {
+  //   // workSections.innerHTML += `
+  // // <li><a href='' class='skill' title='${language}'>${language}</a></li>
+  // // `;
+  // });
```
Comment (Ismailco) [inline js/menu.js]: - [ ] Kindly make sure to remove commented code from your code base so it looks clean and easy to navigate

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi Team, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/menu.js (hunk 2 above)
Comment (IvonneBenitesRodriguez) [reply js/menu.js]: Hello @Ismailco 👋🏾
Thank you for your kind suggestions, I appreciate it,
I corrected all the changes you suggested, 
thanks again,  🌷
Best regards,

**Review by mikethreels — APPROVED** [review]
Hi Ivonne, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job contacting the previous PR.s
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**